### ISSUE TITLE
Reduce number of requests in HOLDER assertion building

### ISSUE THREAD

**Issue by Kailai-Wang** [litentry/litentry-parachain#1655]
### Context

See https://github.com/litentry/litentry-parachain/blob/a5a0917c059a56e27c5941511783f6e643cccea8/tee-worker/litentry/core/assertion-build/src/a4.rs#L164-L168

According to achainable, mixed network type won't be supported. However, it's possible to return "paged" or "sorted" responses for different time durations in one go. This will dramatically reduce the number of needed HTTP requests.

---
:heavy_check_mark: Please set appropriate **labels** and **assignees** if applicable.

**Comment by arieljbirnbaum** [comment]
I take it we should do #1654 first though? :3

**Comment by Kailai-Wang** [comment]
> I take it we should do #1654 first though? :3

Yes that one needs to be done first. This issue mainly includes some communication work

**Comment by arieljbirnbaum** [comment]
#1932 is bumping the number of dates to test (per [network,address] pair) from 7 to 14; will the achainable API still be happy with that or will the requests be too big? :thinking:

**Comment by Kailai-Wang** [comment]
Deprecated - not relevant anymore
